requirements.txt

```diff
@@ -4,6 +4,6 @@ jsonschema>=4.6,<5.0
 libcomps>=0.1.20.post1,<0.2
 productmd~=1.33.0
 pulpcore>=3.17,<3.25
-PyGObject>=3.40,<3.46
+PyGObject>=3.40,<3.45
```

Comment:
It seems like we do not install pygobject with pip (i can only assume it's preinstalled from rpm).
And any upper bound whatsoever will conflict with the upper bounds constraints.
At this point i think we need to exempt this library from the bounds calculator altogether. This also sadly means we never test any version other than the one provided in the image.